svg/core/breadboard/led-rgb-4pin-anode-leg.svg

```diff
@@ -23,16 +23,16 @@
 		<path fill="#8C8C8C" d="M20.786,26.625c-1.12-1.127-2.391-2.236-2.391-3.428s0-5.186,0-5.186h-2.16v5.313
 			c0,1.784,1.535,3.329,3.019,4.822c1.101,1.104,2.346,2.274,2.346,3.373v1.022v0h2.16v-0v-1.185
 			C23.76,29.618,22.248,28.097,20.786,26.625z"/>
-		<path opacity="0.5" fill="#E6E6E6" d="M19.133,13.176V7.184C19.133,3.216,15.916,0,11.948,0
+		<path id="color_path1" opacity="0.5" fill="#E6E6E6" d="M19.133,13.176V7.184C19.133,3.216,15.916,0,11.948,0
 			C7.982,0,4.765,3.216,4.765,7.184v13.834c1.5,1.674,4.153,2.789,7.184,2.789c4.687,0,8.48-2.658,8.48-5.938v-1.547
 			C20.432,15.167,19.95,14.089,19.133,13.176z"/>
-		<path opacity="0.9" fill="#D1D1D1" d="M19.133,13.176v3.146c0,2.777-3.217,5.029-7.185,5.029
+		<path id="color_path2" opacity="0.9" fill="#E6E6E6" d="M19.133,13.176v3.146c0,2.777-3.217,5.029-7.185,5.029
 			c-3.966,0-7.184-2.252-7.184-5.029v3.148v1.547c1.5,1.672,4.153,2.789,7.184,2.789c4.687,0,8.48-2.658,8.48-5.938v-1.546
 			C20.432,15.167,19.95,14.089,19.133,13.176z"/>
 		<path opacity="0.24" d="M19.133,13.176v3.146c0,2.777-3.217,5.029-7.185,5.029
 			c-3.966,0-7.184-2.252-7.184-5.029v3.148c1.5,1.674,4.153,2.789,7.184,2.789c4.687,0,8.48-2.658,8.48-5.938
 			C20.432,15.167,19.95,14.089,19.133,13.176z"/>
-		<ellipse opacity="0.5" fill="#CCCCCC" cx="11.95" cy="16.323" rx="7.183" ry="5.029"/>
+		<ellipse id="color_path2" opacity="0.5" fill="#CCCCCC" cx="11.95" cy="16.323" rx="7.183" ry="5.029"/>
```

Comment:
The id color_path2 is used twice. Was this intended?

Reply:
No, this was a mistake. If the id starts by "color_" them it will change by the simulator code based on the results of the simulation. We can change this to "color_path3". Does this cause a bug? Simulator worked properly

Reply:
This kind of things at least have bug potential. Instead of a single item, the xml parser might return a list. 
This might be some kind of todo. Instead of a magic id, we could use data fields (similar to javascript patterns) to mark dynamic svg content. (The alternative of using CSS I think is overkill, and would probably be unreliable since CSS parsing is hard)

Reply:
Yes, it would be nice to have an easy way to define dynamic svg properties. Will you fix this or should I commit the change to this PR?